egs/thchs30/s5/path.sh

```diff
@@ -0,0 +1,18 @@
+#Change the following settings to match your computing env,
+#so that appropriate (CPU/GPU) Kaldi commands are visible
+
+export KALDI_ROOT=/work3/zxw/kaldi-2016-1-21
```

Comment:
It would be perhaps generally more useful to have the KALDI_ROOT set up in the same way as in other recipes, i.e. three levels up?